demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -891,3 +891,36 @@ def test_check_for_subplaybook_usages(self, repo):
 
         playbook_data = playbook.yml.read_dict()
         assert playbook_data['tasks']['1']['task']['playbookId'] == "my-sub-playbook"
+
+    def test_set_fromversion_six_new_contributor_pack_no_fromversion(self, pack):
+        """
+        Given
+            - An integration from new contributed pack, with no fromversion key at yml
+        When
+            - Run format command
+        Then
+            - Ensure that the integration fromversion is set to 6.0.0
+        """
+        pack.pack_metadata.update({'support': 'partner', 'currentVersion': '1.0.0'})
+        integration = pack.create_integration()
+        bs = BaseUpdate(input=integration.yml.path)
+        bs.set_fromVersion()
+        assert bs.data['fromversion'] == '6.0.0'
+
+    def test_set_fromversion_six_new_contributor_pack(self, pack):
+        """
+        Given
+            - An integration from new contributed pack with fromversion key at yml
+        When
+            - Run format command
+        Then
+            - Ensure that the integration fromversion is set to 6.0.0
+        """
+        pack.pack_metadata.update({'support': 'partner', 'currentVersion': '1.0.0'})
+        script = pack.create_script(yml={'fromversion': '5.0.0'})
+        playbook = pack.create_playbook(yml={'fromversion': '5.0.0'})
+        integration = pack.create_integration(yml={'fromversion': '5.0.0'})
```

Comment:
If the integration has fromversion 5.0.0 we change it to 6.0.0? I am not sure that we want this. Only integration that has fromversion 5.5.0 need to stay like this (we have several like this in our repo)